exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/common/HashTable.java

```diff
@@ -98,6 +98,10 @@ void setup(HashTableConfig htConfig, BufferAllocator allocator, VectorContainer
    */
   int probeForKey(int incomingRowIdx, int hashCode) throws SchemaChangeException;
 
+  int getNum(int currentIndex);
```

Comment:
Please rename this method to clarify that it holds the count of records for a specific key and add JavaDoc.